packages/@aws-cdk/aws-ecs/test/container-definition.test.ts

```diff
@@ -1758,6 +1758,8 @@ describe('container definition', () => {
       const linuxParameters = new ecs.LinuxParameters(stack, 'LinuxParameters', {
         initProcessEnabled: true,
         sharedMemorySize: 1024,
+        maxSwap: 5e3,
```

Comment:
All of these tests need to verify that the correct values are in the resulting template in the THEN block of the test.

Reply:
I added expected values in the templates for all the involved tests.